.github/workflows/pr-checks.yaml

```diff
@@ -29,3 +29,36 @@ jobs:
           fetch-depth: 0
       - name: Check if dockerimage build is working
         run: docker build -f ./Dockerfile .
+  opa_policies_unittest:
+    name: opa_policies_unittest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install dependency packages
+        run: |
+         sudo apt-get install jq
+         sudo curl -L -o /usr/bin/opa https://openpolicyagent.org/downloads/v0.39.0/opa_linux_amd64_static
+         sudo chmod 755 /usr/bin/opa
+
+      - name: Run unittests for policies
+        run: |
+          /usr/bin/opa test policies unittests unittests/test_data -c > /tmp/policy_unittest_result.json
+
+      - name: Check unittests result
+        run: |
```

Comment:
There is another PR for integration tests #8 . @dirgim are we planning to run unit tests as part of CI or locally?

Reply:
I think we can run both the unit tests and the integration tests as part of the CI. I know there's some overlap, but the increased coverage makes up for it at this time.

Reply:
Okay sure.

Reply:
Thanks. We agree to have unittests and integration in CI.